src/Round.js

```diff
@@ -1,12 +1,24 @@
 const data = require('./data');
 const prototypeQuestions = data.prototypeData;
 const util = require('./util');
-const card = require('./Card');
-const deck = require('./Deck');
-const turn = require('./Turn');
+const Turn = require('./Turn');
 
 class Round {
-  constructor() {};
+  constructor(deck) {
+    this.deck = deck;
+    this.turns = 1;
+    this.currentCard = this.deck.cards[this.turns - 1];    
```

Comment:
You may need to make a method that updates the current card, that way you can just set it to the first card in the deck and increment it up by incrementing the turn. If what you have works, then don't mess with it, but I don't know if it will update the current card unless you specifically tell JS to update it.